docs/serverless/dashboards/data-quality-dash.mdx

```diff
@@ -76,8 +75,8 @@ You can share data quality results to help track your team's remediation efforts
 
 **Export results for one index**:
 
-1. Expand an index that has at least one incompatible field by clicking the arrow to the left of its **Result**.
-1. From the **Summary** or **Incompatible fields** tabs, select **Add to new case** to open a new <DocLink slug="/serverless/security/cases-overview">case</DocLink>, or click **Copy to clipboard** to copy a Markdown report to your clipboard.
+1. View check details for an index that has at least one incompatible field by clicking the **View check details** button under **Actions**.
```

Comment:
The first part of this sentence sounds a bit awkward. It might be because "check details" is being used as a noun. Would something like this work:

```suggestion
1. View details for a checked index that has at least one incompatible field by clicking the **View check details** button under **Actions**.
```